daemon/create_unix.go

```diff
@@ -51,7 +52,7 @@ func (daemon *Daemon) createContainerPlatformSpecificSettings(container *contain
 			return err
 		}
 
-		if err := label.Relabel(v.Path(), container.MountLabel, true); err != nil {
+		if err := label.Relabel(v.Path(), container.MountLabel, true); err != nil && err != syscall.ENOTSUP {
```

Comment:
Outside of nfs, is there ever a time when we'd get a ENOTSUP and want to return that as an error?

Reply:
Relabelling of volumes is our attempt to change the builtin labels to something that the container can use.  If the volume is not relabeled in the worse case, the container will fail to be able to use the volume and throw a permission denied error and generate AVCs.

Which is better then failing, when the policy might allow the container to use the unrelabeled content, as it would with NFS.

I believe this is in upstream docker now, to ignore ENOSUP.  Other relabelling code in this repo ignore the ENOSUP error.